frontend/viewer/src/lib/entry-editor/NewEntryDialog.svelte

```diff
@@ -51,12 +68,19 @@
   }
 </script>
 
-<Dialog bind:open on:close={onClosing} {loading} persistent={loading}>
+<Dialog bind:open on:close={onClosing} {loading} persistent={loading} classes={{backdrop: 'overflow-hidden'}}>
```

Comment:
Is this change related to getting rid of the scroll bar?

Reply:
yes, but it didn't work and I forgot to remove it